src/cdk/tree/control/nested-tree-control.ts

```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import {Observable} from 'rxjs/Observable';
-import {first} from 'rxjs/operators';
+import {first} from 'rxjs/operators/first';
```

Comment:
[https://github.com/angular/material2/pull/8352](url)
according to this PR "take(1)" may be better than "first" operator